src/test/java/org/jenkinsci/plugins/docker/workflow/DockerDSLTest.java

```diff
@@ -146,22 +145,6 @@ private static void grep(File dir, String text, String prefix, Set<String> match
         });
     }
 
-    @Issue("JENKINS-37987")
-    @Test public void entrypoint() {
-        story.addStep(new Statement() {
-            @Override public void evaluate() throws Throwable {
-                assumeDocker();
-                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
-                p.setDefinition(new CpsFlowDefinition(
-                        "docker.image('maven:latest').inside {\n" +
```

Comment:
So, what is the behavior of `maven:latest` today? Simply breaking it does not seem like a viable option, as this image is widely used. Is there an updated version which works well? Or an option which can be passed to `image`?

Reply:
maven just works, official docker image validation process do include check for this recommendation for entrypoint to accept an alternative command.
(https://github.com/carlossg/docker-maven/blob/master/jdk-8/mvn-entrypoint.sh#L39)

Reply:
> maven just works

Then why do you need to delete this test case?

Reply:
this test case was removed as I reverted commit 95df0eef05d5e25b4477d22f2d231257667f5586. It can be re-introduced.